### DISCUSSION TITLE
This Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by tesaabi** [vercel/community#6523] [Help]
### Summary

Im working on my flask app with a machine learning model that includes a html file that will be redirected from the path "/". The production deployment is successful but i got this error bellow and 1 other error bellow it too on the web page. How to solve it? please help ... I've seen the learn more page but it wasnt very helpful, please helpp

"Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unuused-build-settings"

> and also got this error on the web, so i cannot run my web page because it was stuck here
This Serverless Function has crashed.

Your connection is working correctly.

Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED

> on the log i got this
LAMBDA_WARNING: Unhandled exception. The most likely cause is an issue in the function code. However, in rare cases, a Lambda runtime update can cause unexpected function behavior. For functions using managed runtimes, runtime updates can be triggered by a function change, or can be applied automatically. To determine if the runtime has been updated, check the runtime version in the INIT_START log entry. If this error correlates with a change in the runtime version, you may be able to mitigate this error by temporarily rolling back to the previous runtime version. For more information, see https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html
[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': No module named 'flask'
Traceback (most recent call last):

### Example

https://obesitas-2x9v70240-tesalonika-abigails-projects.vercel.app/

### Steps to Reproduce

.

**Comment by dooropening** [comment]
i met the same problem, and not solved.i got this 
LAMBDA_WARNING: Unhandled exception. The most likely cause is an issue in the function code. However, in rare cases, a Lambda runtime update can cause unexpected function behavior. For functions using managed runtimes, runtime updates can be triggered by a function change, or can be applied automatically. To determine if the runtime has been updated, check the runtime version in the INIT_START log entry. If this error correlates with a change in the runtime version, you may be able to mitigate this error by temporarily rolling back to the previous runtime version. For more information, see https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html
[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': No module named 'attribute'
i used python + flask, project runs ok on my local machine.
